cli/src/main/scala/bloop/cli/util/OsLibc.scala

```diff
@@ -40,12 +40,15 @@ object OsLibc {
       if (foundMusl)
         Some(true)
       else {
-        val inLib = os.list(os.Path("/lib")).map(_.last)
+        val libPath = os.Path("/lib")
+        val inLib = if (libPath.toIO.exists()) os.list(libPath).map(_.last) else Nil
```

Comment:
A small suggestion:
```suggestion
        val libPath = Some(os.root / "lib").filter(os.exists)
        val inLib = libPath.flatMap(os.list).map(_.last)
```

Reply:
I modified it to Seq instead of Some because I would have to do `toIterable` otherwise, same below